src/nodes/chunk_append/chunk_append.h

```diff
@@ -16,6 +16,7 @@ typedef struct ChunkAppendPath
 	CustomPath cpath;
 	bool startup_exclusion;
 	bool runtime_exclusion;
```

Comment:
So this variable is now a flag for "parent exclusion", and it used to be "chunk-by-chunk exclusion", which is now `runtime_exclude_chunk_constraints`?

Let's add comments about this, and rename the old variable to be specific as well, e.g. `runtime_exclude_by_parent_clauses`.  Otherwise it's easy to miss some usages that relied on the old meaning.

Reply:
👍 refactored stuff to hopefully be more clear